components/divider/stories/divider.stories.js

```diff
@@ -45,5 +48,105 @@ export default {
 	},
 };
 
+/**
+ * The default size for divider is medium.
+ */
 export const Default = Template.bind({});
 Default.args = {};
+Default.parameters = {
+	docs: {
+		story: {
+			height: "auto",
+		}
+	},
+};
+
+/*
+ * Stories for the MDX "Docs" only.
+ * "storyName" refers to the display name/heading for a component
+ */
+const AllDividerSizes = (args) => html`
```

Comment:
I'd like to move the Variant templates into the template.js as an export instead of keeping markup in the stories.js files.